ambry-account/src/test/java/com/github/ambry/account/HelixAccountServiceTest.java

```diff
@@ -650,19 +685,15 @@ public void testDisableBackgroundUpdater() throws Exception {
    *                            received by one {@link Consumer}.
    */
   private void assertAccountUpdateConsumers(Set<Account> expectedAccounts, int expectedNumberOfConsumers,
-      List<Collection<Account>> accountsInConsumers) {
+      List<Collection<Account>> accountsInConsumers) throws Exception {
     assertEquals("Wrong number of consumers", expectedNumberOfConsumers, accountsInConsumers.size());
     for (Collection<Account> accounts : accountsInConsumers) {
       assertEquals("Wrong number of updated accounts received by consumers", expectedAccounts.size(), accounts.size());
       for (Account account : accounts) {
-        assertTrue("Account should be received by the consumers but not.", expectedAccounts.contains(account));
-      }
-      try {
-        accounts.add(Account.UNKNOWN_ACCOUNT);
-        fail("Should have thrown");
-      } catch (UnsupportedOperationException e) {
-        // expected
+        assertTrue("Account should be received by the consumers but not.",
```

Comment:
not your change but this message looks incomplete. Should it read something like "Account update not received by consumers?"